debezium-sink/src/main/java/io/debezium/bindings/kafka/KafkaDebeziumSinkRecord.java

```diff
@@ -5,30 +5,63 @@
  */
 package io.debezium.bindings.kafka;
 
-import java.util.List;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
 import java.util.Set;
 
-import org.apache.kafka.connect.data.Field;
 import org.apache.kafka.connect.data.Schema;
 import org.apache.kafka.connect.data.SchemaBuilder;
 import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.data.Values;
 import org.apache.kafka.connect.errors.ConnectException;
 import org.apache.kafka.connect.header.Header;
 import org.apache.kafka.connect.sink.SinkRecord;
 
 import io.debezium.annotation.Immutable;
+import io.debezium.converters.spi.CloudEventsMaker;
 import io.debezium.data.Envelope;
-import io.debezium.sink.DebeziumSinkRecord;
 import io.debezium.sink.SinkConnectorConfig;
+import io.debezium.sink.field.FieldDescriptor;
+import io.debezium.sink.filter.FieldFilterFactory;
 import io.debezium.util.Strings;
 
 @Immutable
-public class KafkaDebeziumSinkRecord implements DebeziumSinkRecord {
+public class KafkaDebeziumSinkRecord extends Values implements DebeziumSinkRecord {
```

Comment:
I would prefer that we drop `Values` here in the class and instead create an inner class that extends `Values`, moving the `inferSchemaFromMap` static method to the inner class, making it a private concern. That avoids exposing all the public static methods that `Values` exposes, which is unnecessary.

Reply:
Good point.